.github/workflows/release.yml

```
@@ -531,14 +542,14 @@ jobs:
           }
 
       - name: Install Linux (Debian package)
-        if: contains(matrix.component.description, 'linux')
+        if: contains(matrix.component.description, 'linux-x64')
         run: |
           debpath=$(find ./*.deb)
           sudo apt install $debpath
           "${{ matrix.component.command }}" configure
 
       - name: Install Linux (tarball)
-        if: contains(matrix.component.description, 'linux')
+        if: contains(matrix.component.description, 'linux-x64')
```

Let's be clear that we're only validating the linux-x64 artifacts here:

```suggestion
      - name: Install Linux x64 (Debian package)
        if: contains(matrix.component.description, 'linux-x64')
        run: |
          debpath=$(find ./*.deb)
          sudo apt install $debpath
          "${{ matrix.component.command }}" configure

      - name: Install Linux x64 (tarball)
        if: contains(matrix.component.description, 'linux-x64')
```